frontend/components/HelpAndSupport/HelpAndSupport.tsx

```diff
@@ -0,0 +1,175 @@
+import { CloseOutlined, QuestionCircleOutlined } from '@ant-design/icons';
+import { Button, Card, Flex, message, Typography } from 'antd';
+import { useCallback, useEffect, useState } from 'react';
+
+import { DeploymentStatus } from '@/client';
+import { FAQ_URL, SUPPORT_URL } from '@/constants';
+import { UNICODE_SYMBOLS } from '@/constants/unicode';
+import { PageState } from '@/enums';
+import { useBalance, usePageState, useServices } from '@/hooks';
+import { useElectronApi } from '@/hooks/useElectronApi';
+import { useMasterSafe } from '@/hooks/useMasterSafe';
+import { useStore } from '@/hooks/useStore';
+
+import { CardTitle } from '../common/CardTitle';
+import { CardSection } from '../styled/CardSection';
+
+const { Title, Paragraph } = Typography;
+
+const SettingsTitle = () => (
+  <CardTitle
+    title={
+      <Flex gap={10}>
+        <QuestionCircleOutlined />
+        Help & support
+      </Flex>
+    }
+  />
+);
+
+export const HelpAndSupport = () => {
+  const { goto } = usePageState();
+  const { saveLogs } = useElectronApi();
+
+  const { storeState } = useStore();
+  const {
+    serviceStatus,
+    services,
+    hasInitialLoaded: isServiceLoaded,
+  } = useServices();
+  const {
+    isBalanceLoaded,
+    totalEthBalance,
+    totalOlasBalance,
+    wallets,
+    walletBalances,
+    totalOlasStakedBalance,
+  } = useBalance();
+
+  const {
+    backupSafeAddress,
+    masterSafeAddress,
+    masterEoaAddress,
+    masterSafeOwners,
+  } = useMasterSafe();
+
+  const [isLoading, setIsLoading] = useState(false);
+  const [canSaveLogs, setCanSafeLogs] = useState(false);
+
+  const onSaveLogs = useCallback(() => {
+    setIsLoading(true);
+    setCanSafeLogs(true);
+  }, []);
+
+  const handleSaveLogs = useCallback(() => {
+    return saveLogs?.({
+      store: storeState,
+      debugData: {
+        services: {
+          services:
+            services?.map((item) => ({
+              ...item,
+              keys: item.keys.map((key) => key.address),
```

Comment:
I only safe `key.address` here - to avoid storing `private_key`. there's also a `ledger` fields in keys, lmk if this could be useful in logs